### ISSUE TITLE
[BUG][TYPO3 8.5-dev] flux:field.inline.fal not working

### ISSUE THREAD

**Issue by monosize** [FluidTYPO3/flux#1248]
Testing with PR #1247, which displays the correct container.

After selecting an image error 500 is displayed.

form the the sys_log 
```#1446996319: Configuration retrieved from FlexForm is incomplete or not of type "inline". | UnexpectedValueException thrown in file /typo3/sysext/backend/Classes/Controller/FormInlineAjaxController.php in line 921```

Testing with this configuration:
```html
<f:section name="Configuration">
  <flux:form id="testelement">
    <flux:field.inline.fal name="images" multiple="false" minItems="1" maxItems="1" showThumbs="1"
       allowedExtensions="jpg,jpeg,png"
       foreignMatchFields="{fieldname:'image'}"
       foreignUnique="uid_local"
      />
  </flux:form>
</f:section>```

With TYPO3 7.6 there a no problems.

**Comment by NamelessCoder** [comment]
This looks like it relates to the current problem of DS resolving for new records (which I assume this is about?). Inline configuration cannot be retrieved and passed to the inline field HTML. We plan on adding displayCond to hide Flux fields until the parent record is saved - after which, there should be no problem resolving the right DS.

The problem *may* potentially be solved in TYPO3 8.x, will *probably not* be solved in 7.6 - and we expect can be avoided once we raise our minimum required TYPO3 version to 8 LTS.

**Comment by pwodaniilea** [comment]
@NamelessCoder 

Hello. 

So there is currently no fix for this on TYPO3 8.5?

**Comment by NamelessCoder** [comment]
There is no fix for this in neither 7.6 LTS nor 8.x, sorry.

**Comment by BenjaminBeck** [comment]
Hello @NamelessCoder 
i think @pwodaniilea also meant that he can´t add a FAL image to an already saved fluid-content-element.
I do have the same issue.

When debugging the mentioned 500er error i found that the function ``parseDataStructureByIdentifierPreProcess`` does not return the DS although it could.

I made a patch:
https://github.com/FluidTYPO3/flux/pull/1309
